pplns-with-job-declaration.tex

```diff
@@ -21,174 +21,193 @@
 \begin{remark}
-	Note that
-	\[\sum_{j=1}^N score_d(s_i) = 1\] and that this score depends only on difficulty.
+	Note that \[\sum_{j=1}^N score_d(s_i) = 1\] depends only on difficulty.
 \end{remark}
-\[\sum_{j=1}^N score_d(s_i) = 1\] and that this score depends only on difficulty.
+
 \subsection{Fee-based score}
 Suppose we want to score the $i$-th share $s_i$, and suppose that this share belongs to a slice $S$ that contains all the shares from the $k_1$-th to the $k_2$-th.
-For the $i$-th share, call $\bar d_i = score_d(s_i)$ its score difficulty (calculated as above) and $f_i$ the fees of the share (which depends on the transactions chosen in the template). For this share, define the score relative to the fee
+For the $i$-th share, $\bar d_i = score_d(s_i)$ is the difficulty-based score (calculated as above) and $f_i$ the fees of the share (which depends on the transactions chosen in the template). We define the score relative to the fee
 \[
-score_f(s_i) = \frac{\bar d_if_i}{\sum_{j=k_1}^{k_2}\bar d_j f_j}.
+	score_f(s_i) = \frac{\bar d_if_i}{\sum_{j=k_1}^{k_2}\bar d_j f_j}.
 \]
 \begin{remark}\label{properties-diff-based-score}
-\begin{enumerate} 
-\item As remaked earlier, the dependence on difficulty is intentional. Otherwise, if there are two shares with the same fee, the one with lower difficulty is paid the same as the other, while requiring less computational effort to be produced.
-\item Suppose that the slice $S$ contains $n$ shares, all of the same difficulty score. Then, if $f_i = f_{max}$ is the max fee of every share in the slice $S$, then the payout for this share is at least $score_f(s_i) \ge 1/n$. This sentence is proven by contraddiction. So, assume that 
-\[score_f(s_i) < 1/n.\]
-Since all the shares have the same difficulty score, we have 
-\[\frac{\bar d_if_{max}}{\sum_j d_j f_j} = \frac{f_{max}}{\sum_j f_j}.\]It follows that
-\[ \frac{f_{max}}{\sum_j f_j} <\frac{1} {n}.\]
-So,
-\[ 1 = \frac{\sum_k f_k}{\sum_j f_j} = \sum_k \frac {f_k}{\sum_j f_j} \le \sum_k \frac{f_{max}}{\sum_j f_j}<\sum_k\frac{1}{n} = 1\]
-which is impossible.
-\item if there is $c>0$ such that $f_i = cf_j$ for some $s_i, s_j$ two shares in the slice $S$, then $payout_f(s_i) = c \cdot payout_f(s_j)$. This means that if a miner becomes $c$ times more capable of finding a more profitable template, his shares will be paid proportionally.
-\item similarly, if $\bar d_i = c \bar d_j$, then $payout_f(s_i) = c \cdot payout_f(s_j)$ this similarly represents the fact that a miner is paid proportionally with respect his hashpower.
-\item similarly to difficulty score, if the slice $S$ contains shares from $k_1$-th to $k_2$-th, we have \[\sum_{i=k_1}^{k_2} score_f(s_i) = 1. \] \label{addstoone}
-\end{enumerate}
+	\begin{enumerate}
+		\item The dependence on difficulty-based scoring is intentional so two shares with the same fee also account for computational work needed.
```

Comment:
I would change it like this
" As mentioned above, it is important that so two shares with the same fee also account for computational work needed.  Without this, two shares with the same fee, the one with lower difficulty is paid the same as the other, while requiring less computational effort to be produced. The most natural way to fix this is to construct fee-based score on top of difficulty-based score."